sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -1,10 +1,10 @@
-# Azure Form Recognizer client library for Python
+# Azure Document Intelligence (previously Form Recognizer) client library for Python
```

Comment:
When Text Analytics rebranded under the Language service, we continued to call it the "text analytics client library". Anywhere we referred to the client library, we used "text analytics", anywhere we referred to the service, we used "language service". Maybe this could work for FR?